### PR TITLE
update to 4.0.1

### PR DESCRIPTION
**Pull request by cbouss**
watchdog 4.0.1

**Destination channel:** main

### Links

- https://anaconda.atlassian.net/browse/PKG-4954
- https://github.com/gorakhargosh/watchdog/tree/v4.0.1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,8 +32,15 @@ requirements:
...
+    - tests/
   requires:
     - pip
+    - pytest
+    - pytest-timeout
+    - pytest-cov
+    - flaky
   imports:
     - watchdog
     - watchdog.observers
```
Comment (Jrice1317) [inline recipe/meta.yaml:46]: Don't think it's a blocker, but I'm not seeing `watchdog.observers.fsevents`. Was this possibly replaced with `_watchdog_fsevents`?

Comment (Jrice1317) [reply recipe/meta.yaml:46]: Also, why the first underscore in `_watchdog_fsevents`?

**Review by Jrice1317 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:46]: `watchdog.observers.fsevents` is https://github.com/gorakhargosh/watchdog/blob/v4.0.1/src/watchdog/observers/fsevents.py

Comment (skupr-anaconda) [reply recipe/meta.yaml:46]: Seems` _watchdog_fsevents` for `osx` is https://github.com/gorakhargosh/watchdog/blob/v4.0.1/setup.py#L44

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**